src/Entity/Admin.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\AdminRepository;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+/**
+ * @ORM\Entity(repositoryClass=AdminRepository::class)
```

Comment:
I generally prefer to name tables explicitely (`@ORM\Table(name="admins")`: it ensures changes in defualt naming won't affect the project and it allows us to use plural names, which I tend to prefer